src/react-app.js

```diff
@@ -162,27 +183,28 @@ function renderReactApp({
             whiteSpace: 'nowrap',
           }}
         >
-          {extraCredit.map(({extraCreditTitle, id}, index) => (
+          {files.map(({id, filename}) => (
             <Tab key={id} css={{display: 'flex', alignItems: 'center'}}>
-              {React.createElement(getExtraIcon(index), {
+              {React.createElement(getExtraIcon(filename), {
                 size: 20,
                 style: {marginRight: 5},
               })}
-              <span>{extraCreditTitle}</span>
+              <span>{filename}</span>
```

Comment:
Every file of a given type *should* have a unique title and I think it would be better than the filename.

Reply:
while they have a unique `extraCreditTitle`, it can be really long in the example and I was hoping to move it to the next line to avoid horizontal scrolling + make it easier to find the source file by showing filename :)

![title vs filename](https://user-images.githubusercontent.com/1087670/95383107-f58d9c00-08ea-11eb-8456-dfedb4fac2d7.png)